api/src/controllers/communityUserController.js

```diff
@@ -82,22 +83,38 @@ const followCommunity = async (req, res) => {
 
 const getAllMembers = async (req, res) => {
   try {
-    const data = await db.CommunityUser.findAll(
+    const pageSize = 8
```

Comment:
make pageSize dynamic with default may be 8 or 9 When screen resize send pageSize based on number of item we show for that screensize * 3 rows

Reply:
made pageSize dynamic and implemented in community member page.